packages/cli/src/commands/preview/server/setup.ts

```diff
@@ -18,6 +18,8 @@ import { template } from "lodash";
 import { getNodeModulesDirsFrom } from "../../util/getNodeModulesDirsFrom";
 
 export const COMPONENT_FILE_REGEXP = /^[^\s-]+\.[tj]sx$/; // no spaces, .jsx or .tsx
+export const DOT_MAILING_IGNORE_REGEXP =
+  /__test__|generator_templates|yarn-error.log|src\/commands|src\/index\.ts$|src\/dev\.js$|\.mailing$|\.next|node_modules$|^cypress/;
```

Comment:
@psugihara was there a reason we added `src\/commands` in this branch?  The e2e tests fail because the manifest and feManifest ejs templates are in a subdir of src\/commands

Reply:
nah, i don't think there is a reason